frontend/controller/actions/identity.js

```diff
@@ -343,6 +343,15 @@ export default (sbp('sbp/selectors/register', {
       sbp('state/vuex/commit', 'login', loginAttributes)
       await sbp('chelonia/storeSecretKeys', () => transientSecretKeys)
 
+      if (!state) {
+        // Make sure we don't unsubscribe from our own identity contract
+        // Note that this should be done _after_ calling
+        // `chelonia/storeSecretKeys`: If the following line results in
+        // syncing the identity contract and fetching events, the secret keys
+        // for processing them will not be available otherwise.
+        await sbp('chelonia/contract/retain', identityContractID)
```

Comment:
- Does it really make sense to call retain on the identity contract here?
- Is it possible for the retain count to be incremented too high this way?
- Should this code maybe be moved inside of a `try` block, like right after line 373?

Reply:
> Does it really make sense to call retain on the identity contract here?

Yes. There is no state at that point, so the identity contract must be retained.

> Is it possible for the retain count to be incremented too high this way?

It's not, as this only happens when there's no state

> Should this code maybe be moved inside of a try block, like right after line 373?

Sure.